==== triton/include/triton/Dialect/Triton/IR/TritonOps.td#15 - /google/src/cloud/akuegel/mlir_d790a217a746ff63190d30669674fce1bc0a4723_1690966528/triton/include/triton/Dialect/Triton/IR/TritonOps.td ====
# action=edit type=text
--- triton/include/triton/Dialect/Triton/IR/TritonOps.td	2023-07-07 10:10:42.000000000 -0700
+++ triton/include/triton/Dialect/Triton/IR/TritonOps.td	2023-08-02 02:27:52.000000000 -0700
@@ -627,6 +627,9 @@
     operand_range getArgOperands() {
       return {arg_operand_begin(), arg_operand_end()};
     }
+    MutableOperandRange getArgOperandsMutable() {
+      return getOperandsMutable();
+    }
 
     operand_iterator arg_operand_begin() { return operand_begin(); }
     operand_iterator arg_operand_end() { return operand_end(); }
